repositories/books.js

```diff
@@ -0,0 +1,41 @@
+const { bookDB } = require('../db');
+
+function addBookWithAuthorTitleAndPages(author, title, pages) {
```

Comment:
_too_ descriptive naming here))
`addBookWithAuthorTitleAndPages(author, title, pages)` should be 
`addBook(author, title, pages) `or `addBook({author, title, pages})`